srcpkgs/sbctl/patches/keyusage.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/keys.go b/keys.go
```

Comment:
please add a note or link explaining this patch

Reply:
Done.